src/modules/services_manager_service.rb

```diff
@@ -440,6 +422,37 @@ def toggle(service)
       enabled(service) ? disable(service) : enable(service)
     end
 
+    # Sets start_mode for a service (in memory only, use save())
```

Comment:
can we use for this array of ServiceConfiguration ??? If something missing we can adapt it, but in general it looks like this has same goal.

Reply:
no, I mean use ServiceConfiguration to hold that changes and then write it at once. One ServiceConfiguration per service.

Reply:
Well, that's what I meant (I explained myself badly). @joseivanlopez and me want to get rid of the hash with all those values inside and `ServiceConfiguration` could fit (you may need the hash to index `ServiceConfiguration` instances, but that's a different topic). I would do not change it in this PR, but I am adding a task in the PBI.

On the other hand, I guess `ServiceConfiguration` should rely on the new `SystemService` class, right?

Reply:
Yes. Goal of ServiceConfiguration is to hold configuration and read/write it in proper time. And also handle multiple services at once. Not sure if in the end we don't split it into two classes?

Reply:
At this point I am not sure at all whether we will need `ServiceConfiguration` and `SystemService` at all. But I think that we can re-evaluate later (for the time being, they make sense).